src/schemas/28-1900/schema.js

```diff
@@ -52,6 +53,9 @@ let schema = {
       type: 'number',
       minimum: 0
     },
+    dd214: Object.assign({}, originalDefinitions.files, {
```

Comment:
Let's change the name of this field to something like `dischargeDocuments`. dd214 is kind of misleading and we may want to fix that on VIC in the future, too.

Reply:
Agreed- I'll make a ticket to update it for VIC